server/Dockerfile

```diff
@@ -5,9 +5,6 @@ FROM ${BASE_BUILDER_IMAGE} AS server-builder
 
 WORKDIR /home/server-builder
 
-COPY Makefile ./
```

Comment:
This isn't actually needed in the ui-server version of the Dockerfile, GRPC files are already built before we sync so we don't need any utils to build them.